runtime/gc_glue_java/ArrayletObjectModel.hpp

```diff
@@ -931,6 +931,30 @@ class GC_ArrayletObjectModel : public GC_ArrayletObjectModelBase
 	 */
 	void AssertArrayPtrIsIndexable(J9IndexableObject *arrayPtr);
 
+	/**
+	 * Asserts that the dataAddr field of the indexable object is correct.
+	 *
+	 * @param arrayPtr      Pointer to the indexable object
+	 * @param dataAddr      Pointer to indexable object data address
+	 */
+	MMINLINE bool
+	isCorrectDataAddrContiguous(J9IndexableObject *arrayPtr, void *dataAddr)
+	{
+		return dataAddr == ((void *)((uintptr_t)arrayPtr + contiguousHeaderSize()));
```

Comment:
Redundant parentheses, it could be:
```
  return dataAddr == (void *)((uintptr_t)arrayPtr + contiguousHeaderSize());
```
Same comment for line 955.